test/Microsoft.DotNet.Docker.Tests/DotNetImageType.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.DotNet.Docker.Tests
+{
+    public enum DotNetImageType
+    {
+        SDK,
```

Comment:
Minor - should these be alphabetical?

Reply:
I'm following the principle that the default enum value should be the most common value.  Since the SDK image is the "latest" image, having the SDK be the default value makes most sense to me.